[vtk] Use our drake_vendor prefix for glew vendoring

This situation is unique to Drake, so we won't to upstream this patch.

--- ThirdParty/glew/vtkglew/include/GL/vtk_glew_mangle.h
+++ ThirdParty/glew/vtkglew/include/GL/vtk_glew_mangle.h
@@ -1,13 +1,22 @@
 #ifndef vtk_glew_mangle_h
 #define vtk_glew_mangle_h
 
-#define glewGetErrorString vtkglew_glewGetErrorString
-#define glewGetExtension vtkglew_glewGetExtension
-#define glewGetString vtkglew_glewGetString
-#define glewInit vtkglew_glewInit
-#define glewIsSupported vtkglew_glewIsSupported
-#define glxewGetExtension vtkglew_glxewGetExtension
-#define glxewInit vtkglew_glxewInit
-#define glxewIsSupported vtkglew_glxewIsSupported
+#define glewGetErrorString drake_vendor_glewGetErrorString
+#define glewGetExtension drake_vendor_glewGetExtension
+#define glewGetString drake_vendor_glewGetString
+#define glewInit drake_vendor_glewInit
+#define glewIsSupported drake_vendor_glewIsSupported
+#define glxewGetExtension drake_vendor_glxewGetExtension
+#define glxewInit drake_vendor_glxewInit
+#define glxewIsSupported drake_vendor_glxewIsSupported
+
+#define vtkglew_glewGetErrorString drake_vendor_glewGetErrorString
+#define vtkglew_glewGetExtension drake_vendor_glewGetExtension
+#define vtkglew_glewGetString drake_vendor_glewGetString
+#define vtkglew_glewInit drake_vendor_glewInit
+#define vtkglew_glewIsSupported drake_vendor_glewIsSupported
+#define vtkglew_glxewGetExtension drake_vendor_glxewGetExtension
+#define vtkglew_glxewInit drake_vendor_glxewInit
+#define vtkglew_glxewIsSupported drake_vendor_glxewIsSupported
 
 #endif
